src/main.cpp

```diff
@@ -853,7 +862,7 @@ int main(int argc, char** argv) {
           fwrite(w->getFinalBuf(),1,w->size(),f);
           fclose(f);
         } else {
-          reportError(fmt::sprintf(_("could not open file! (%s)"),e.getLastError()));
+          reportError(fmt::sprintf(_("could not open file! (%s)"),cmdOutName.c_str()));
```

Comment:
The text in %s is intended to be an error. Revert.

Reply:
I believe that `strerror(errno)` is closer to your intent, see note below.